src/components/ThreeUpText.tsx

```diff
@@ -27,7 +27,7 @@ const ItemTitle = ({ text, small }: ItemTitle): ReactElement => {
     return (
         <Tag
```

Comment:
I can't tag the line number in this file, but I cc'd you on a comment in Figma for the Hashicorp design about a slight change to this component. We need to edit out the TagName conditional to have h4 or h2 depending on the small variant

Reply:
I see, ok take a look now